storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -4227,7 +4253,11 @@ void xtrabackup_backup_func(void) {
   }
 
   if (changed_page_bitmap) {
+    msg("xtrabackup: WARNING: Incremental backup using page bitmap is "
+        "deprecated and will be removed in a future release.\n");
```

Comment:
Add this text as well
Please use --page-tracking instead

Reply:
Also disalow combinations of --page-tracking and change-page tracking

Reply:
for bitmap user doesn't need to specify any option to PXB. It just need to be enabled on the server. 
but to use page-tracking user need to specify --page-tracking.
so if the user has enabled bitmap on server and specify --page-tracking. Page-tracking with take precedence. 
should we error out in such case and suggest user to disable bitmap on server or 
the current behviour is to continue with page-tracking.

Reply:
@rahulmalik87 if we find both page-tracking and changed page bitmap tracking (Redo based), we can print a message to the user here that the user can disable bitmap tracking by using the command "SET GLOBAL innodb_changed_page_tracking=OFF" (check option name).